iphone/Classes/TiUIiOSProxy.m

```diff
@@ -795,5 +802,42 @@ - (id)createFeedbackGenerator:(id)args
 MAKE_SYSTEM_PROP(LARGE_TITLE_DISPLAY_MODE_NEVER, UINavigationItemLargeTitleDisplayModeNever);
 #endif
 
+#ifdef USE_TI_UIWEBVIEW
+
+- (id)createWebViewConfiguration:(id)args
+{
+  return [[[TiUIiOSWebViewConfigurationProxy alloc] _initWithPageContext:[self executionContext] args:args] autorelease];
+}
+
+- (id)createWebViewProcessPool:(id)args
+{
+  return [[[TiUIiOSWebViewProcessPoolProxy alloc] _initWithPageContext:[self executionContext] args:args] autorelease];
+}
+
+MAKE_SYSTEM_PROP(CREDENTIAL_PERSISTENCE_NONE, NSURLCredentialPersistenceNone);
+MAKE_SYSTEM_PROP(CREDENTIAL_PERSISTENCE_FOR_SESSION, NSURLCredentialPersistenceForSession);
+MAKE_SYSTEM_PROP(CREDENTIAL_PERSISTENCE_PERMANENT, NSURLCredentialPersistencePermanent);
+MAKE_SYSTEM_PROP(CREDENTIAL_PERSISTENCE_SYNCHRONIZABLE, NSURLCredentialPersistenceSynchronizable);
+
+MAKE_SYSTEM_UINTEGER(AUDIOVISUAL_MEDIA_TYPE_NONE, WKAudiovisualMediaTypeNone);
```

Comment:
Is this new macro necessary? For example, the above constants for credential persistence also are `NSUInteger` enum values and they use `MAKE_SYSTEM_PROP `

Reply:
This is of type NSUInteger.
MAKE_SYSTEM_PROP(AUDIOVISUAL_MEDIA_TYPE_ALL, WKAudiovisualMediaTypeAll); will give following error message-
"Implicit conversion from 'NSUInteger' (aka 'unsigned long') to 'int' changes value from 18446744073709551615 to -1"
because WKAudiovisualMediaTypeAll = NSUIntegerMax

Reply:
Ah ok, can you rename it to `MAKE_SYSTEM_PROP_UINTEGER` to align with `MAKE_SYSTEM_PROP `?

Reply:
Agreed. Updated.